data-model/SDP-data-model-templates/codicological-unit_sdp-template.json

```diff
@@ -35,8 +35,11 @@
       "note": [
```

Comment:
For some reason I can't add a comment above here...But I just noticed that support (line 10-20) is missing a 'range' field, which would function like the one in writing_and_hands to specify which folios are a given support type. This field could be optional, a null value meaning that the entirety of the unit is the specified support type. But if there are multiple in the same unit, I think we'd want some kind of indication of which folios are which.